google/cloud/spanner_v1/database.py

```diff
@@ -108,11 +108,20 @@ class Database(object):
                  passed, the database will construct an instance of
                  :class:`~google.cloud.spanner_v1.pool.BurstyPool`.
 
+    :type logging_enabled: boolean
+    :parama logging_enabled: (Optional) Represents whether the database
+                    has logging enabled or not. Default is True
+
```

Comment:
In database close() can we add code to close all bg tasks and a test for the same.

Reply:
database does not have a close() method in Python. Every pool has a clear method which deletes all the session, added stopCleaningLongRunningSessions call in the same method.  But that method is not referenced any where .